mathics/builtin/inout.py

```diff
@@ -42,6 +47,37 @@
 SymbolTableDepth = Symbol("TableDepth")
 
 
+class PrintForms_(Predefined):
+    """
+    <dl>
+    <dt>'$PrintForms'
```

Comment:
Old habits die hard.

Reply:
I should have added this to the consistency tests.